### ISSUE TITLE
Electron: Open a Quarto doc fails on MacOS due to old location of share directory

### ISSUE THREAD

**Issue by ronblum** [rstudio/rstudio#10919]
### System details

    RStudio Edition : Desktop (Electron)
    RStudio Version : 2022.06.0-daily-272 (Electron)
    OS Version      : MacOS 12.3
    R Version       : 4.1.3

### Steps to reproduce the problem

This is on MacOS only.
1. Don't install the Quarto CLI from Quarto.org. If it's installed, remove it.
2. In RStudio Desktop, open a new Quarto document.
3. Try to run the bundled `quarto` from the command line: `/Applications/RStudio.app/Contents/Resources/app/bin/quarto/bin/quarto `

### Describe the problem in detail

1. The file fails to open, with this pop up instead:
<img width="255" alt="Screen Shot 2022-04-06 at 1 10 00 PM" src="https://user-images.githubusercontent.com/31009952/162031415-fd56d87b-e2c6-434e-84d4-5df8d9592d84.png">
2. The command line call returns the following:

```
/Applications/RStudio.app/Contents/Resources/app/bin/quarto/bin/quarto: line 77: cd: /Applications/RStudio.app/Contents/Resources/app/bin/quarto/bin/../share: No such file or directory
```

### Describe the behavior you expected

The Quarto document should open, and calling `quarto` should return nothing instead of an error.

This works correctly in the Qt version of Spotted Wakerobin. Note that even though `quarto` has been moved, the `share` directory is still in the old place (I renamed the applications in order to have Electron and Qt side by side):

```
ronblum@Rons-Busy-Bee ~ % find /Applications/RStudio-2022.06.0-daily-272-Electron.app -name share
/Applications/RStudio-2022.06.0-daily-272-Electron.app/Contents/Resources/quarto/share
ronblum@Rons-Busy-Bee ~ % find /Applications/RStudio-2022.06.0-daily-272-Electron.app -name quarto
/Applications/RStudio-2022.06.0-daily-272-Electron.app/Contents/Resources/app/bin/quarto
/Applications/RStudio-2022.06.0-daily-272-Electron.app/Contents/Resources/app/bin/quarto/bin/quarto
/Applications/RStudio-2022.06.0-daily-272-Electron.app/Contents/Resources/quarto
ronblum@Rons-Busy-Bee ~ % 
ronblum@Rons-Busy-Bee ~ % find /Applications/RStudio-2022.06.0-daily-272-OG.app -name share      
/Applications/RStudio-2022.06.0-daily-272-OG.app/Contents/Resources/quarto/share
ronblum@Rons-Busy-Bee ~ % find /Applications/RStudio-2022.06.0-daily-272-OG.app -name quarto
/Applications/RStudio-2022.06.0-daily-272-OG.app/Contents/MacOS/quarto
/Applications/RStudio-2022.06.0-daily-272-OG.app/Contents/MacOS/quarto/bin/quarto
/Applications/RStudio-2022.06.0-daily-272-OG.app/Contents/Resources/quarto
```

This also works fine on Red Hat 8.3:
```
[vmware@localhost ~]$ find /usr/lib/rstudio/ -name share
/usr/lib/rstudio/resources/app/bin/quarto/share
[vmware@localhost ~]$ find /usr/lib/rstudio/ -name quarto
/usr/lib/rstudio/resources/app/bin/quarto
/usr/lib/rstudio/resources/app/bin/quarto/bin/quarto
```

- [ ] I have read the guide for [submitting good bug reports](https://github.com/rstudio/rstudio/wiki/Writing-Good-Bug-Reports).
- [ ] I have installed the latest version of RStudio, and confirmed that the issue still persists.
- [ ] If I am reporting an RStudio crash, I have included a [diagnostics report](https://support.rstudio.com/hc/en-us/articles/200321257-Running-a-Diagnostics-Report).
- [ ] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.

**Comment by timtmok** [comment]
This should be fixed in daily #278. It was only an issue for Mac and the fix didn't get built for a while since there were so many intermittent failures that it didn't kick off the Mac build.

See #10871

**Comment by ronblum** [comment]
[Noting that it's a duplicate of #10871.]
